docs/ja/ai-integration/overview.md

```
@@ -1,23 +1,21 @@
 
-| エンドポイント | フォーマット | 説明 |
+| エンドポイント | 形式 | 説明 |
 |---------------|------|------|
 | `GET /llms.txt` | Markdown (llms.txt) | LLM 向け API ドキュメント |
 | `GET /tools.json` | JSON | Claude Tool Use / OpenAI Function Calling 互換スキーマ |
 | `GET /.well-known/ai-plugin.json` | JSON | AI プラグインマニフェスト |
 | `GET /openapi.json` | JSON | OpenAPI 3.0 仕様 |
-| `GET /api.json` | JSON | API リファレンス |
-
-## Tool Use スキーマ (`/tools.json`)
+| `GET /api.json` | JSON | API リファレンス |## ツール使用スキーマ (`/tools.json`)
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)